sdk/keyvault/arm-keyvault/package.json

```diff
@@ -42,7 +42,7 @@
     "esm/**/*.js.map",
     "esm/**/*.d.ts",
     "esm/**/*.d.ts.map",
-    "lib/**/*.ts",
+    "src/**/*.ts",
```

Comment:
This needs to be reflected in the generator as well. CC @daschult.

Reply:
https://github.com/Azure/autorest.typescript/pull/439